data/luminr-co.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Luminr",
+    "establishedAt": 1697548800000,
+    "location": "Kaldhara, Kathmandu",
+    "website": "https://luminr.co//",
+    "email": "business@luminr.co",
+    "phone": "+977-9860050391",
```

Comment:
If this place has a landline number, please add it here.

Reply:
There is no landline right now, unfortunately, when it's added I will update it accordingly.